public/pages/sessions/session6/introduction.html

```diff
@@ -182,7 +182,7 @@ <h2>Previous showcase</h2>
           const student = childSnapshot.val();
           const userId = childSnapshot.key;
 
-          if (student.profile && student.profile.siteFlag) {
+          if (student.profile && student.profile.siteFlag === "yes" ) {
```

Comment:
would it make more sense to set siteFlag to true/false rather than a string?

Reply:
Totally!

Reply:
updated in the code to use a boolean instead of a string